prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.ts

```diff
@@ -105,6 +109,12 @@ export class EnrolleeTableComponent implements OnInit, OnChanges {
     );
   }
 
+  // Check to see if we want to filter/map the status
+  // Conditions: enrollee is a paper enrollee and the status is one of [Under Review, Editable] mapped to [Incomplete, Complete] respectively
+  public filterPaperEnrolmentStatus(enrollee: EnrolleeListViewModel): boolean {
```

Comment:
We probably want to find a better name for this method, when you see filter our mind first jumps to a method that takes in a list and returns a filtered down list.

With proper naming we can get rid of the comments above. Since comments are just code we have to maintain.

Reply:
to be honest you can just move the logic from the template into a public method
``` getEnrolmentStatus(row: EnrolleeListViewModel): string ```

Reply:
Then use that method to localize the logic and when to use the paper enrolment map